lib/bpl/transformation/shadowing.rb

```diff
@@ -151,7 +151,7 @@ def run! program
           params[:public_in_value].each do |p|
             p.names.each do |x|
               decl.append_children(:specifications,
-                bpl("requires #{shadow_eq x};"))
+                bpl("requires #{shadow_eq bpl x};"))
```

Comment:
Careful here: `shadow_eq bpl x` is equivalent to `shadow_eq(bpl,x)` not `shadow_eq(bpl(x))`.